eval/tricks.rkt

```diff
@@ -8,54 +8,57 @@
          "../ops/all.rkt"
          "machine.rkt")
 
-(provide get-ampls
+(provide get-bounds
          get-slack)
 
-(define (get-slack)
-  (match (*sampling-iteration*)
-    [1 512]
-    [2 1024]
-    [3 2048]
-    [4 4096]
-    [5 8192]))
+(define (get-slack [iter (*sampling-iteration*)])
+  (match iter
+    [0 0]
+    [_ (* (expt 2 (- iter 1)) 512)]))
```

Comment:
Why?

Reply:
Ah, just what if a user will enter `6` iterations instead of `5`